rowboat/plugins/utilities.py

```diff
@@ -299,15 +315,15 @@ def seen(self, event, user):
                 Message.author_id == user.id
             ).order_by(Message.timestamp.desc()).limit(1).get()
         except Message.DoesNotExist:
-            return event.msg.reply(u"I've never seen {}".format(user))
+            return event.msg.reply(u"I've never seen {}".format(user)).after(5).delete()
 
         event.msg.reply(u'I last saw {} {} ago (at {})'.format(
             user,
             humanize.naturaldelta(datetime.utcnow() - msg.timestamp),
-            msg.timestamp
+            msg.timestamp.strftime("%d-%b-%y @ %H:%M")
```

Comment:
consider adding `GMT` or `UTC` so people don't think it's localized?

Reply:
also consider moving timestamp format to constants so we don't reuse it over and over?

Reply:
At the moment I just replaced everything from isoformat to strftime to clean up the formatting